src/game_battlealgorithm.cpp

```diff
@@ -1171,6 +1173,12 @@ bool Game_BattleAlgorithm::Skill::Execute() {
 			}
 		}
 
+		//If resurrected and no HP selected, the effect value is a percentage:
```

Comment:
That's is not visible in the diff but in the else-block is another "int effect" which shadows the effect declaration, I assume this is wrong.

missing whitespace after //

Reply:
more "static analysis": The else-codepath is for "non-healing", so that if with "IsRevived" isn't taken then but why aren't these 4 new lines then not simply put into the healing if-block?

Reply:
The effect code was confusing and ugly. I've cleaned it up.

I also agree with you that `IsRevived()` belongs in the upper healing block.